### PR TITLE
[bitnami/phpmyadmin] Add support for ingressClassName

### PR DESCRIPTION
**Pull request by markwooff**
**Description of the change**

Adds support for `ingress.ingressClassName`

**Benefits**

Replaces a deprecated feature of using `kubernetes.io/ingress.class` annotation to determine ingress class.

**Possible drawbacks**

None known.

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/phpmyadmin/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: phpmyadmin
 sources:
   - https://github.com/bitnami/bitnami-docker-phpmyadmin
   - https://www.phpmyadmin.net/
-version: 8.4.0
+version: 8.4.1
```
Comment (javsalgar) [inline bitnami/phpmyadmin/Chart.yaml:32]: Could you bump the minor version instead? It is adding a non-breaking feature

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thank you so much for the PR! Minor change requested

File: bitnami/phpmyadmin/Chart.yaml (hunk 1 above)
Comment (markwooff) [reply bitnami/phpmyadmin/Chart.yaml]: No problem, I've bumped the minor version for this PR.

**Review by javsalgar — APPROVED** [review]
Thank you so much! LGTM

**Pull request merged**